.github/workflows/publish.yml

```diff
@@ -1,14 +1,8 @@
 name: Publish
 
 on:
-  push:
-    tags: ["v[0-9]+.[0-9]+.[0-9]+"]
-  workflow_dispatch:
-    inputs:
-      tag:
-        description: The existing tag to publish
-        type: string
-        required: true
+  release:
```

Comment:
@oscar-izval to make things easier across all CI workflows, I believe it would be preferable to have `semantic-release` create a `prerelease` first, as this would allow us to perform a final check before pushing the button. On the other hand, I believe this would allow us to work around the problem with the `workflow_dispatch` you found. What are your thoughts?

Reply:
Sounds good! I'd love to see a fully automatic process, but I agree with you that we may not be there yet